sync/sync.go

```diff
@@ -239,6 +247,7 @@ Register:
 		return fmt.Errorf("CheckAndResetFailedRebuild failed, error: %s", err.Error())
 	}
 
+	s.SetPreload(false)
 	logrus.Infof("Adding replica %s in WO mode", replicaAddress)
 	_, err = t.client.CreateReplica(replicaAddress)
 	if err != nil {
```

Comment:
we can reset it to true and add a comment saying 'after operations like create, reload, preload will be reset to its default value i.e., true.. same is followed at other places where these operations are performed'